web/src/components/dashboard/BusinessStructurePrompt.tsx

```diff
@@ -19,20 +22,31 @@ export const BusinessStructurePrompt = (): ReactElement => {
     >
       <div className={isDesktopAndUp ? "flex flex-row flex-align-start" : ""}>
         <div className={isDesktopAndUp ? "flex-fill margin-right-2" : "margin-bottom-2"}>
-          <Content>{Config.businessStructurePrompt.notCompletedTaskPrompt}</Content>
-        </div>
-        <div className={isDesktopAndUp ? "flex-auto" : ""}>
-          <PrimaryButton
-            isColor="primary"
-            isRightMarginRemoved
-            isFullWidthOnDesktop
-            onClick={(): void => {
-              router.push(ROUTES.businessStructureTask);
-            }}
-          >
-            {Config.businessStructurePrompt.buttonText}
-          </PrimaryButton>
+          {router.asPath === ROUTES.businessStructureTask ? (
```

Comment:
Same comment as before - we should look up the url from the roadmap instead of hardcoding it.

That'll have to be a change to the test as well.